dawn/src/dawn/Optimizer/PassSetBoundaryCondition.cpp

```diff
@@ -58,7 +58,7 @@ static iir::Extents analyzeStencilExtents(const std::unique_ptr<iir::Stencil>& s
   return fullExtents;
 }
 
-enum FieldType { FT_NotOriginal = -1 };
+enum class FieldType { NotOriginal = -1 };
```

Comment:
does this make sense?

Reply:
We need to cast it to `int` anyway, so why not? `static_cast<int>(FieldType::NotOriginal)` is more descriptive than `(int)NotOriginal`

Reply:
I will change this back right now!